storage/postgres/broker.go

```diff
@@ -96,6 +96,29 @@ func (bs *brokerStorage) GetAll() ([]types.Broker, error) {
 	return brokers, nil
 }
 
+func (bs *brokerStorage) GetByName(names []string) ([]types.Broker, error) {
```

Comment:
The code for platforms and brokers seems almost the same. Is it possible to extract the common part? Maybe in the controllers as well if possible.